AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/KMSKeystoreOperations.dfy

```diff
@@ -896,4 +985,117 @@ module {:options "/functionSyntax:4" } KMSKeystoreOperations {
     && |decryptHistory.output.value.Plaintext.value| == (Structure.BKC_DIGEST_LENGTH + Structure.AES_256_LENGTH) as int
   }
 
+  method packAndCallKMS(
```

Comment:
Suggestion: I do not think this method should be in KMS Keystore Operations.
Yes, it calls KMS; but it does a number of other things as well.
```suggestion
  //TODO-HV-2-M4: Move to HVUtils
  method packAndCallKMS(
```